src/components/RadioGroup/Indicator.tsx

```diff
@@ -0,0 +1,33 @@
+import * as RadioGroup from "@radix-ui/react-radio-group";
+import { forwardRef } from "react";
+import { PropsWithBox } from "../Box";
+import { indicator } from "./Item.css";
+
+export type RadioGroupIndicatorProps = PropsWithBox<{
+  disabled?: boolean;
+  asChild?: boolean;
+}>;
+
+export const RadioGroupIndicator = forwardRef<
+  HTMLDivElement,
+  RadioGroupIndicatorProps
+>(({ disabled, asChild }, ref) => {
```

Comment:
**Question:** Won't types be wrong here?

I mean you can use any type from `Box` but we need to spread the rest of `...props` on svg component 🤔